docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -71,34 +71,16 @@ Ensure that your environment includes the following:
 
 <Tabs>
 <TabItem label="Authenticated Proxy">
-Starting the local database proxy with the `--tunnel` flag will create an
-authenticated tunnel that you can use to connect to your database instances.
-You won't need to configure any credentials when connecting to this tunnel.
 
-Here is an example on how to start the proxy:
-
-```bash
-# Start the local proxy.
-$ tsh proxy db --tunnel <database-name>
-Started authenticated tunnel for the <engine> database "<database-name>" in cluster "<cluster-name>" on 127.0.0.1:62652.
-```
-
-You can optionally specify the database name and the user to use by default
-when connecting to the database:
-
-```code
-$ tsh proxy db --db-user=my-database-user --db-name=my-schema --tunnel <database-name>
-```
-
-Now, you can connect to the address the proxy command returns. In our example it
-is `127.0.0.1:62652`.
+(!docs/pages/includes/database-access/tsh-db-tunnel.mdx!)
 
 </TabItem>
 <TabItem label="TLS routing">
-If you're using Teleport in [TLS routing](../management/operations/tls-routing.mdx)
-mode where each database protocol is multiplexed on the same web proxy port, use
-the following command to start a local TLS proxy your GUI database client will
-be connecting to:
+
+(!docs/pages/includes/determinetls.mdx!)
```

Comment:
Should we include the instructions to determine whether TLS Routing is enabled *before* determining which approach to use to connect to the database, i.e., the authenticated proxy, `tsh proxy db`, or a direct database connection?

Reply:
introduced in the beginning now